extensions/2.0/Vendor/EXT_geopose/README.md

```diff
@@ -0,0 +1,8 @@
+# EXT_geopose
```

Comment:
Are we sure this changelog is necessary?  It is not customary.

Reply:
We have been following this convention (ex: [3DTILES_implicit_tiling](https://github.com/CesiumGS/3d-tiles/blob/3d-tiles-next/extensions/3DTILES_implicit_tiling/README.md), [3DTILES_metadata](https://github.com/CesiumGS/3d-tiles/blob/3d-tiles-next/extensions/3DTILES_metadata/README.md), [EXT_feature_metadata](https://github.com/CesiumGS/glTF/blob/3d-tiles-next/extensions/2.0/Vendor/EXT_feature_metadata/README.md)) to provide direct references and changelogs for specifications that may depend on these extensions.